cmd/maya-apiserver/spc-actions/cstorpool_create.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2017 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cstorpool
+
+import (
+	"errors"
+	"github.com/golang/glog"
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	"github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	"github.com/openebs/maya/pkg/storagepool"
+	"fmt"
+)
+
+func CreateCstorpool(spcGot *apis.StoragePoolClaim) (error) {
+
+	// Check wether the spc object has been processed for cstor pool creation
+	if(spcGot.Status.Phase=="online"){
+		return errors.New("Pool Already Exsists")
```

Comment:
1. Typo for "Exists".
2. Log message should start with upper case and remaining words should be in lower-case.
3. Rephrase the message similar to "CStor pool already exists since the status on SPC object is Online."

Reply:
Sure